cpmpy/expressions/globalconstraints.py

```diff
@@ -494,6 +698,25 @@ class Xor(GlobalConstraint):
         The 'xor' exclusive-or constraint
     """
 
+    """
+    Main functionalities:
+    The Xor class is a subclass of the GlobalConstraint class and represents the exclusive-or constraint. It takes a list 
+    of Boolean arguments and enforces that exactly one of them is true.
```

Comment:
This is not quite correct: "xor" is true iff the number of true variables in the given list is odd (with only two arguments, xor is indeed equivalent to 'exactly one is true', but we allow more).